src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -125,6 +125,9 @@ export class TableComponent implements AfterViewInit, OnInit, OnChanges, OnDestr
   @Input()
   hasDetails = false;
 
+  @Input()
+  showMenu = true;
```

Comment:
```suggestion
  showToolbar = true;
```

Would that be better? I found it kinda confusing.
Referring to: https://carbondesignsystem.com/components/data-table/usage/#anatomy

Reply:
Thanks @Pegonzal, I checked that doc and in the context of carbon datatable, those are called inline actions. the toolbar is something that holds the search bar and filters along with the primary action.
https://carbondesignsystem.com/components/data-table/usage/#inline-actions